src/tests/class_tests/openms/source/MzIdentMLFile_test.cpp

```diff
@@ -72,8 +72,7 @@ START_SECTION(void load(const String& filename, std::vector<ProteinIdentificatio
 {
   std::vector<ProteinIdentification> protein_ids;
   std::vector<PeptideIdentification> peptide_ids;
-  std::vector<String> fm;
-  fm.push_back("Carbamidomethyl (C)");
+  std::vector<String> fm{"Carbamidomethyl (C)", "Xlink:DTSSP[88] (Protein N-term)"};
```

Comment:
@enetz Eugen does this make sense

Reply:
In the tests further below (line 96) only one of the fixed modifications is checked with `fm.back()`. That was fine before, since there was only one modification. Now it might be a better idea to check both of them.
Otherwise this PR looks fine to me.

Reply:
Wasn't there some other problem that prevented us to include the protein terminal ones? I vaguely recall there were some discussions quite a while ago.